Tools/QedTablesUtils/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+########
+# Common argument parser library
+########
+set(arg_parser_cpp "Source/ArgParser/QedTablesArgParser.cpp")
+set(arg_parser_h   "Source/ArgParser/QedTablesArgParser.H")
+
+add_library( QedToolsArgParser ${arg_parser_cpp} ${arg_parser_h} )
```

Comment:
Is this library used anywhere else?

If you like, generalize it and make it part of the ABLASTR utils?